packages/story-editor/src/elements/shape/display.js

```diff
@@ -42,6 +42,8 @@ const Element = styled.div`
   ${elementFillContent}
   ${elementWithBackgroundColor}
   ${elementWithBorder}
+  ${({ backdropBlur }) =>
+    backdropBlur && `backdrop-filter: blur(${backdropBlur}px)`}
```

Comment:
Hm, was this specified anywhere or is it just because it looks good?

Reply:
Figma says so:

![image](https://user-images.githubusercontent.com/21036224/129739180-e33d4e5a-7185-4cd9-9aa9-89290a7a8ea2.png)